horreum-web/src/domain/tests/FoldersTree.tsx

```diff
@@ -56,7 +56,8 @@ export default function FoldersTree(props: FoldersTreeProps) {
             data={[root]}
             activeItems={activeItem ? [activeItem] : []}
             onSelect={(_, item) => {
-                props.onChange(item.id || "")
+                if (item.id)
+                    props.onChange(item.id)
```

Comment:
This change means users will not be able to unselect a folder (go back to the default folder) by clicking in the UI. I agree that it is annoying to have the folder change when trying to close the tree, perhaps the tree should close when a folder is selected rather than preventing the change when clicking the top level?

Reply:
I think we can handle the tree functionality when we refactor the UI layout, e.g. : https://github.com/Hyperfoil/Horreum/pull/1290#issuecomment-1933469772

Reply:
This change prevents a call to the backend when the tree is opened.